### PR TITLE
add pause button

### PR DESCRIPTION
**Pull request by DusanSacha**
Implementing #34

### CODE REVIEW

File: src/components/Game.js [hunk 1]
```diff
@@ -68,6 +76,14 @@ class Game extends Component {
     });
   };
 
+  handleSnackbarClose = reason => {
+    //if (reason !== 'clickaway' || this.state.gameState === GAME_STARTED) {
```
Comment (ovidiubute) [inline src/components/Game.js]: You can remove the comments.

File: src/components/Menu.js [hunk 2]
```diff
@@ -7,6 +7,11 @@ import Avatar from 'material-ui/Avatar';
 import Alarm from 'material-ui/svg-icons/action/alarm';
 import Moves from 'material-ui/svg-icons/action/compare-arrows';
 import Replay from 'material-ui/svg-icons/av/replay';
+import Pause from 'material-ui/svg-icons/av/pause';
+import Play from 'material-ui/svg-icons/av/play-arrow';
+import { GAME_STARTED, GAME_PAUSED } from '../lib/game-status';
+import PropTypes from 'prop-types';
+// import New from 'material-ui/svg-icons/action/power-settings-new';
```
Comment (ovidiubute) [inline src/components/Menu.js]: Not used, so you can remove this.

File: src/components/Menu.js [hunk 3]
```diff
@@ -52,6 +64,26 @@ class Menu extends Component {
...
+            className="menuButton"
+            label="New game"
+            onTouchTap={onResetClick}
+            icon={<New className="menuIcon" />}
+          />
+        */
+          }
+          <RaisedButton
+            className="menuButton"
+            label={gameState === GAME_PAUSED ? 'Continue' : 'Pause Game'}
```
Comment (ovidiubute) [inline src/components/Menu.js]: "Pause" is enough, I think, it should be obvious you're pausing the game, so it doesn't need to be in the label.

**Comment by DusanSacha** [comment]
Thanks for the review! I applied the changes.

**Comment by ovidiubute** [comment]
Nice job, man! 🍰

**Pull request merged**